simp_le.py

```diff
@@ -1649,19 +1650,19 @@ def _single_path_stats(path):
 
     def test_it(self):
         webroot = os.path.join(os.getcwd(), 'public_html')
-        args = ('--server %s --tos_sha256 %s -f account_key.json '
-                '-f key.pem -f full.pem -d le.wtf:%s' % (
-                    self.SERVER, self.TOS_SHA256, webroot))
+        args = ["simp_le", "-v", "--server", (self.SERVER), "--tos_sha256",
```

Comment:
I think changing the name of the variable to `cmd` would make this a bit more clear, as now this includes the full command, not just the args. Same thing elsewhere.